packages/arb-provider-ethers/src/lib/l2bridge.ts

```diff
@@ -34,3 +36,30 @@ export async function withdrawEth(
     value,
   })
 }
+
+interface AddressIndexMemo {
+  [address: string]: number
+}
+
+export const getAddressIndex = (() => {
```

Comment:
Took a minute for me to figure out how the closure worked with the `arbSys` var, but this is super smart